packages/pn-personafisica-webapp/src/components/Deleghe/DelegationsElements.tsx

```diff
@@ -73,13 +73,20 @@ export const Menu = (props: any) => {
   return (
     <>
       <IconButton
+        id="delegation-menu-icon"
```

Comment:
The component is used multiple times into Delegation page. It used for each delegation so this static id is not unique